### ISSUE TITLE
Alan Wake (108710)

### ISSUE THREAD

**Issue by tr37ion** [ValveSoftware/Proton#156]
Arch Linux
Kernel 4.18.3
Proton 3.7
Nvidia GTX 1070 - nvidia-vulkan 396.51
KDE Plasma 5.13.4
KDE Framework 5.49.0
Qt 5.11.1
GFX settings MAX

Alan Wake works without problems as tested so far.

**Comment by hadbabits** [comment]
It's a shame most of us won't be able to play it; their music licenses expired and now they don't sell it anywhere :(

**Comment by LeJimster** [comment]
Edit: game now works for me since proton was updated.

[My System Specs](https://gist.github.com/LeJimster/7751f48f10bf0082158bace47dec7430)

**Comment by dubigrasu** [comment]
Runs OK for me (performance wise) but I can see some light related glitches (some square-ish black shadows in the very beginning of the game). Tried various settings, still the same.
SteamOS, AMD 7970.

**Comment by LeJimster** [comment]
Since todays Proton update Alan Wake is now working for me.

**Comment by libcg** [comment]
Alan Wake is running here, but the performance is really low (I believe it's a DirectX 10 game, that's not yet enabled on DXVK) and the mouse control is very bad (could be related to #147).

**Comment by rlac** [comment]
I did a quick 10 minute test with Proton 3.7-3.

No audio in cut scenes, adding -forcestereo to launch options fixed that. Also a few minor graphical issues (some lighting & blur effects didn't look quite right), but nothing that was game breaking for what I tested.

OS: Ubuntu 18.04
GPU: GeForce GTX 970
Drivers: NVIDIA 396.54

**Comment by tr37ion** [comment]
@LeJimster Not sure, why you ask for DXVK. Proton is the DXVK implementation + Wineprefix for each game already. That is why we focus on Proton here not DXVK.

**Comment by LeJimster** [comment]
@tr37ion because the game currently uses Wines OpenGL and not DXVK.  I've just raised a bug on the DXVK bug tracker, but it might be that the game doesn't use DX11.

Update:  Seems like its a DX9 game only.

**Comment by andrebadaia** [comment]
Confirmed.

Distro: Linux Mint 19 Tara
Processor: AMD Ryzen 7 1700
GPU: GTX 1070, Nvidia proprietary driver 396.54
RAM: 32GB
Kernel (custom) - 4.17.14 (optimized for Ryzen)
Resolution: 1920x1080
Proton 3.7-3

**Comment by mimattr** [comment]
Ubuntu 16.04, NVIDIA GTX 970, Proton 3.7-5 Beta:

[Steam System Information](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)

Test | Result
-- | --
Singleplayer | Working as expected
Local Co-op | _N/A_
Online Multiplayer | _N/A_

Configuration | ...
-- | --
Input | Steam Controller
Display | 1920x1080
Fullscreen | Yes
Preset | Medium
VSync | Off
API | DX9

**Comment by pedrofleck** [comment]
Played for like 30 minutes and it worked ok. The performance was ok (~40 in high and ~55 in very low, it looks like a CPU bottleneck even though I have a i5 4670K that should run it 60fps+), there are graphical glitches like the grass (other transparent elements may have it too) - https://i.imgur.com/4EBUgzA.jpg - and I can't change the gamma using the in-game calibration. The game didn't crash and I used alt-tab to check other applications, changed a lot of graphics configs and Steam overlay. My GPU is a GTX 1060 6GB overclocked.

**Comment by vanyasem** [comment]
The game crashes at launch on nvidia GPUs: https://paste.ubuntu.com/p/McYtrRVZdp/
System info: https://gist.github.com/vanyasem/f787469ec8c50cdd3728340ef4c73269
Proton log: [steam-108710.log](https://github.com/ValveSoftware/Proton/files/2363258/steam-108710.log)

The log suggests that it's a NVAPI issue, tracked here: https://github.com/ValveSoftware/Proton/issues/165. The workaround mentioned [here](https://github.com/ValveSoftware/Proton/issues/350#issuecomment-419514874) fixes the crash issue.

However, using any graphics preset other than "very low" results in a black screen or glitchy textures (probably related to the fact that nvapi is a stub). I consider this game not being whitelist-ready as of now.

![20180908141620_1](https://user-images.githubusercontent.com/13350458/45253574-3e707080-b372-11e8-85d6-2ec87d5e8e12.jpg)

**Comment by pedrofleck** [comment]
> The game crashes at launch on nvidia GPUs: https://paste.ubuntu.com/p/McYtrRVZdp/
> System info: https://gist.github.com/vanyasem/f787469ec8c50cdd3728340ef4c73269
> Proton log: [steam-108710.log](https://github.com/ValveSoftware/Proton/files/2363258/steam-108710.log)
> 
> The log suggests that it's a NVAPI issue, tracked here: #165. The workaround mentioned [here](https://github.com/ValveSoftware/Proton/issues/350#issuecomment-419514874) fixes the crash issue.
> 
> However, using any graphics preset other than "very low" results in a black screen or glitchy textures (probably related to the fact that nvapi is a stub). I consider this game not being whitelist-ready as of now.
> 
> ![20180908141620_1](https://user-images.githubusercontent.com/13350458/45253574-3e707080-b372-11e8-85d6-2ec87d5e8e12.jpg)

The crash with Nvidia GPUs wasn't present with Proton 3.7-3 and these graphical issues aren't present with me, only blocky grass textures

**Comment by pchome** [comment]
The only thing in game `nvapi` used for is Nvidia 3D Vision (I suppose).
So if you don't expect this feature to work, than forget about `nvapi` (Proton 3.7-3 don't contain it).
<details>
<summary>NvAPI related logs</summary>

    0009:trace:nvapi:DllMain (0x7b830000, 8, (nil))
    0009:trace:nvapi:DllMain (0x7b830000, 1, (nil))
    0009:trace:nvapi:nvapi_QueryInterface (150e828)
    0009:trace:nvapi:NvAPI_Initialize ()
    0009:trace:nvapi:nvapi_QueryInterface (33c7358c)
    0009:trace:nvapi:nvapi_QueryInterface (593e8644)
    0009:trace:nvapi:nvapi_QueryInterface (be7692ec)
    0009:trace:nvapi:get_thunk_function (be7692ec)
    0009:fixme:nvapi:unimplemented_stub function 0xbe7692ec is unimplemented!
    0009:trace:nvapi:nvapi_QueryInterface (ac7e37f4)
    0009:trace:nvapi:NvAPI_Stereo_CreateHandleFromIUnknown (0x158f58, 0x2ad08ac)
    0009:trace:nvapi:nvapi_QueryInterface (4b708b54)
    0009:trace:nvapi:NvAPI_D3D_GetCurrentSLIState (0x158f58, 0x33db64)
    0009:trace:nvapi:nvapi_QueryInterface (1fb0bc30)
    0009:trace:nvapi:NvAPI_Stereo_IsActivated ((nil), 0x33dccf)
    0009:trace:nvapi:NvAPI_Stereo_IsActivated ((nil), 0x33dc8b)
    ...
  The only thing the game do on my system with `nvapi` further: repeatedly called `NvAPI_Stereo_IsActivate`
  Unimplemented_stub is `NvAPI_Stereo_CreateConfigurationProfileRegistryKey`
</details>

<br/>
Also `nvapi` from wine-staging depends on `d3d11`, which probably will use  DXVK's `d3d11` and `dxgi` dlls (while running Proton), if ` PROTON_USE_WINED3D11` not set.
<br/><br/>
Tested on non-steam #125 , but there should be no big difference.<br/>
Using wine-3.16/wine-staging-3.16 , no matter is `nvapi` present or not, the game works fine using HIGH graphics profile. <br/>No DXVK was set up, since game using dx9.

**Comment by pedrofleck** [comment]
The grass with "blocky" textures seems to be present on Windows too, but it's related with AMD cards and AA or AF being forced by the driver. I don't have theses settings enabled and I'm using a Nvidia card and I have this bug though.
https://community.remedygames.com/forum/games/alan-wake/alan-wake-help-issues/7740-alan-wake-grass-textures-bug-help
https://steamcommunity.com/app/108710/discussions/0/828939978483402754/
https://steamcommunity.com/app/108710/discussions/0/864969953367987372/?l=turkish

**Comment by kisak-valve** [comment]
[REGRESSION] Alan Wake stopped launching in proton 3.16-3

Issue transferred from https://github.com/ValveSoftware/Proton/issues/1815.
@Patola posted on 2018-10-26T10:28:07:

# Compatibility Report
- Alan Wake (appid 108710) launches correctly in all proton 3.7 releases. When being run in 3.16-3, however, it clears the screen for a couple seconds then crashes.

## System Information
- Ubuntu 18.04 64 bits 16 GiB RAM, using LXDE as desktop environment
- GPU: NVIDIA GTX 1070
- Driver/LLVM version: NVIDIA binary driver 396.54.09
- Kernel version: 4.15.0-38-lowlatency
- Proton version: 3.16-3

## I confirm:
- [X] that I HAVE found an existing compatibility report for this game. It should run.
- [X] that I have checked whether there are updates for my system available.

## Symptoms
- Click "Play". If it is the first time, it spends some time installing requisites. Then it clears the screen and enters fullscreen mode. Then it crashes. On versions of proton prior to 3.16, it launches correctly.
- Note: Alan Wake's American Nightmare (a sequel) _works_ with the same setup, either in proton 3.7 or proton 3.16.

[steam-108710.log](https://github.com/ValveSoftware/Proton/files/2518816/steam-108710.log)


## Reproduction
Install Alan Wake and click play.
***
@Patola commented on 2018-10-26T10:41:51

Game runs with launch options "PROTON_NO_ESYNC=1 PROTON_NO_D3D11=1 %command%". But it should run without them since it works that way on 3.7.

**Comment by Patola** [comment]
Hi, I am very sorry, but further tests I made reveal that it works even without the launch options. Thing is, the game needs to be launched a half dozen times until he actually starts. It seems that as I wasn't able to launch at first and tried different options, it finally launched and I confused the options I made with something that was required to play the game.

Having said that, it is really odd that a game needs to be launched repeatedly until it finally "obeys". Computers were meant to be deterministic!

**Comment by Marco-GG** [comment]
There is some glitching/artifacts on illumination. Here you can see the difference between a Windows system and a Linux with proton:
Linux: https://youtu.be/ECagCMGX95Y?t=157
Windows: https://youtu.be/2hIEFTcwtyI?t=165

I got the exact same bugged illumination as is showed on the first video using a RX 580 with Mesa 18.2.2 and Proton 3.16-3.

**Comment by LeJimster** [comment]
@Marco-GG I'm not sure if this will be helpful as a fix for Proton.  But I discovered the same bug when trying to run the game with gallium-nine, which has now been [fixed](https://github.com/iXit/Mesa-3D/issues/316).  It's a real shame Proton doesn't have the ability to enable gallium-nine support, especially with VK9 being a year or two away at a minimum.

**Comment by CSahajdacny** [comment]
Weird lights
![108710_20190104231932_1](https://user-images.githubusercontent.com/45825041/50719187-65685600-1077-11e9-8053-d5d2c3b872b3.png)

And yes, is a DX9 game... VK9 project will be very usefull on Proton.

**Comment by corvusd** [comment]
Hello community and devs,

this game are far to will add to a the white-list. It need resolved at least three bug, I make a list by order of importance:

1) No shafts or volumetric light at all. And this is one of the most characteristic features of the game design.
2) Lawn not show correct the transparency mask if Vsync is activated or it is configured during the game. Need to reset the game without Vsync to show correct, to tricky, but work for now.
3) Some issues whit sound at start or during the game. Some hang out it possible because this.
4) Performance. Bottleneck is important in this game. Is true, a lot of dx9 games in wine have this problem until. Is possible play between 20/30 FPS in most scenarios.

Greetings!

David Gámiz Jiménez

PD: Run in Proton 3.16-8 Beta. 4.2 not work sound in intro or game, not both at same time, maybe Faudio minor issue.

My Hardware:
AMD Phenom(tm) II X6 1055T Processor × 6
8GB RAM DDR3-1333
SSD Sata3 KINGSTON 120Gb / Seagate Sata3 HDD 1TB
Nvidia GeForce GTX 1060 OC 3072MB VRAM DDR5
Ubuntu 16.04 LTS (Last updated) / Unity 7
Driver Nvidia: 415.27 official graphics ppa
Gamepad Xbox360 Wired USB
Steam Gamepad
Webcam Logitech QuickCam E 3500

**Comment by roman-bronis** [comment]
I have no sound in cut-scenes. Any idea how to fix this? Sound in game-play and menus works.

Intel i5 6500
16GB RAM
nVidia GTX 1060 6GB 418.74
Arch x64 5.0.13

**Comment by roman-bronis** [comment]
In case someone reads this, the issue in my case was fixed by changing launch option to:
`%command% -forcesurround`
(even if you have stereo headphones, like I do)
If this ain't helping, according to what I read, -forcestereo is also worth trying.
**EDIT**: too soon. Even with parameter the sound in cut-scenes is not every time. If the game starts with sound on Remedy logo, the sound in cut-scenes works. If not, no sound in cut-scenes. Only thing which helps me is restarting the game till the point when the sound starts to work...

**Comment by LeJimster** [comment]
For the adventurous...  The game is now running pretty well on [D9VK](https://github.com/Joshua-Ashton/d9vk).  You can grab up-to-date binaries [here](https://git.froggi.es/joshua/d9vk/-/jobs).

Extract the 32-bit d3d9.dll into Alan Wakes folder and add this to your Launch Options for the game in steam:
`WINEDLLOVERRIDES="d3d9=n" DXVK_HUD=full %command%`

You can remove the `DXVK_HUD=full` part if you don't want to see the stats, I just find it useful to verify the game is actually running d9vk.

**Comment by roman-bronis** [comment]
With D9VK performance seems better, but I found at least one graphical glitch - in game, when activating special TVs, image is not displayed. Sound issue stays the same... ;)
Proton 4.2.4  
![20190521225528_1](https://user-images.githubusercontent.com/1164597/58132985-fc970a00-7c22-11e9-8997-0406b4284bbc.jpg)
Proton 4.2.4 with D9VK dll  
![20190521225743_1](https://user-images.githubusercontent.com/1164597/58133009-0ae52600-7c23-11e9-8d8f-c8700f1758aa.jpg)

**Comment by kisak-valve** [comment]
Hello @roman-bronis, graphical issues with D9VK should be reported to that project as it is not included in Proton.

**Comment by roman-bronis** [comment]
> Hello @roman-bronis, graphical issues with D9VK should be reported to that project as it is not included in Proton.

Sorry, it wasn't supposed to sound like report of the issue. I meant it like status report for anyone interested in trying D9VK. ;)

**Comment by corvusd** [comment]
@kisak-valve D9VK is in proton yet. These issues now is OK, no?
For other way, the sound in cutscenes is the only real annoying bug for me. Is very strange. Because work in one way not in both. Fmod project can be help here?

**Comment by RussianNeuroMancer** [comment]
> PD: Run in Proton 3.16-8 Beta. 4.2 not work sound in intro or game, not both at same time, maybe Faudio minor issue.

Nobody tried to install XNA instead of FAudio? Maybe this will fix sound issues?

**Comment by AwesamLinux** [comment]
**System Information**
- GPU: RX VEGA 56
- GPU Driver: 19.1.0+aco+git1568912505-b738bf06917-1bionic2 (Valves PPA)
- Kernel version: 5.3.1-050301-lowlatency
- Distro version: Linux Mint 19.2 Tina (64 bit) (Cinnamon)
- Proton version: 4.11-6

**Compatibility Report**

Out of the box there are rendering issues with foliage transparency, and game launches minimized.

But with D9VK (launch option "PROTON_USE_D9VK=1 %command%") the game looks correct, and launched fullscreen like it should.

![aland9vk](https://user-images.githubusercontent.com/43387963/65638074-a9c0cb00-dfe5-11e9-9f1b-a6255f5a9ed4.jpg)
Left is with D9VK[steam-108710-proton-4.11-6.log](https://github.com/ValveSoftware/Proton/files/3654693/steam-108710-proton-4.11-6.log)
[steam-108710-proton-4.11-6-d9vk.log](https://github.com/ValveSoftware/Proton/files/3654694/steam-108710-proton-4.11-6-d9vk.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3654695/system_info.txt)

Using D9VK I have so far not encountered issues. Also I did not hear any audio crackling (that was a problem with some older Proton version I tried before, however it could be fixed with PULSE_LATENCY_MSEC=60).

[steam-108710-proton-4.11-6.log](https://github.com/ValveSoftware/Proton/files/3654698/steam-108710-proton-4.11-6.log)
[steam-108710-proton-4.11-6-d9vk.log](https://github.com/ValveSoftware/Proton/files/3654699/steam-108710-proton-4.11-6-d9vk.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3654700/system_info.txt)

**Comment by corvusd** [comment]
The graphics are work ok, since few months thanks to Josh work in d9vk. But the main issue, at least for me, It is the sound in cinematics or FMV not work at all.

**Comment by CSahajdacny** [comment]
I managed to solve the lack of audio in the cutscenes using PROTON 4.11-11 with 
WINEDLLOVERRIDES="xaudio2_7=**b**,**n**" 

Just in case, I use this for to play:
`PROTON_FORCE_LARGE_ADDRESS_AWARE=1 PROTON_NO_ESYNC=1 WINEDLLOVERRIDES="xaudio2_7=b,n" PROTON_USE_D9VK=1 %command% `

I detect some stuttering in the cutscenes when Vsync setting is ON but the gameplay is smooth.

**Comment by RazrFalcon** [comment]
**System Information**

GPU: 1070 TI
GPU Driver: 440.59
Kernel version: 5.5.6
Distro version: Gentoo Linux x86_64
Proton version: 5.0-4

**Compatibility Report**

No sound out of the box. Heavy shadow flickering during camera movement. Performance is comparable to the Windows.

`WINEDLLOVERRIDES="xaudio2_7=b,n"` fixes the sound in the game, but breaks it in cut scenes.

**Comment by iWeaker** [comment]
@kisak-valve  With the release of Proton 6.3, it stops working in the first "boot", after the first it already works, after each restart of my computer that little detail happens.

It does not happen with Proton 5.13-6 or Proton GE 6.5

GPU 1060 6gb
Kernel: 5.12.0-rc5-144-tkg-cfs
Distro: Arch Linux XFCE 
GPU Driver: 465.19.01 beta

[steam-108710.log](https://github.com/ValveSoftware/Proton/files/6247047/steam-108710.log)

**Comment by kisak-valve** [comment]
Hello @iWeaker4you, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Log hints at an access violation (null pointer?) in winmm which makes this audio related.

**Comment by iWeaker** [comment]
@kisak-valve  https://gist.github.com/iWeaker4you/9753923b4550af777cbac9079aa441bc

**Comment by alasky17** [comment]
@iWeaker4you I tried to repro this and saw an inconsistent crash on launch with both 5.13-6 and 6.3-1.  The crash didn't happen consistently after rebooting the whole rig or with a fresh prefix.  It seems like an inconsistent failure that has been around for a while, not a regression.  Perhaps you are just hitting it more frequently now because of some timing change.

**Comment by iWeaker4you** [comment]
@alasky17  
The problem happens to me regardless of the distro I am using, I currently install Debian in a partition, and the same thing happens, and only with that version of Proton, with the rest, just one click works fine.

**Comment by iWeaker** [comment]
With the latest beta version of 6.3, when wanting to exit the game, it freezes my PC , after a minute I have to force it to exit.
It did not happen before updating to this beta,
Not even in the experimental version
[steam-108710.log](https://github.com/ValveSoftware/Proton/files/6267396/steam-108710.log)

**Comment by Simmonz** [comment]
For me various cut scenes have no audio. Other cut scenes have audio and the game itself has audio.

I added PROTON_LOG=1 %command% to the game's launch and got the following log:

[steam-108710.log](https://github.com/ValveSoftware/Proton/files/6370928/steam-108710.log)


I have checked whether there are updates for my system available and I am up to date

System Information:

Distro: Manjaro

GPU: RX 580 8GB

Driver/LLVM version: Mesa 21.0.2/11.0.1

Kernel version: 5.11.14-1-MANJARO

Proton version: 6.3-2

https://gist.github.com/Simmonz/ec5ce89f07d41a4afe400d425a04ae38

**Comment by CSahajdacny** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/156#issuecomment-826167770

Have you tried with
`WINEDLLOVERRIDES="xaudio2_7=b,n" %command%`

**Comment by Simmonz** [comment]
> > Replying to [#156 (comment)](https://github.com/ValveSoftware/Proton/issues/156#issuecomment-826167770)
> 
> Have you tried with
> `WINEDLLOVERRIDES="xaudio2_7=b,n" %command%`

Thanks for the tip. I tried it with that and so far the cut scenes that were giving me issues all work fine now. I'll come back to report if that changes further into the game.

**Comment by JRDetwiler** [comment]
I'm experiencing crashing upon starting the game, no matter what settings I apply.

Distro: Arch Linux
Kernel: 5.18.6
Proton: Experimental / 7.0-3 / 6.3-8 / 5.13-6 / 5.0-10 / 4.11-13
Launch Options: pretty much every mix of the DLL overrides and large address options. also tried [this](https://github.com/ValveSoftware/Proton/issues/156#issuecomment-569108081).
Hardware: System76 [`gaze16-3050`](https://tech-docs.system76.com/models/gaze16/README.html)

I've checked the integrity of the game files too, and it seems odd that I can't find anyone else with the same issue. Would anyone know what's going on?

**Comment by mtb-xt** [comment]
> I'm experiencing crashing upon starting the game, no matter what settings I apply.

I'm also experiencing this problem, can't get Alan Wake to run at all, also on Arch, 5.18

**Comment by michaelneverwins** [comment]
Alan Wake crashes on launch when I try to run it on my Steam Deck, despite having "verified" status. I've tried Proton 7.0-3 ("selected by Valve testing"), as well as Proton Experimental.

Meanwhile, it works with Proton 7.0-3 on my PC running Linux Mint 20.3, even without the `WINEDLLOVERRIDES="xaudio2_7=b,n" %command%` launch option which I had needed with older Proton versions.

**Comment by mtb-xt** [comment]
> Meanwhile, it works with Proton 7.0-3 on my PC running Linux Mint 20.3

Do you have an Nvidia GPU in your PC? All reports of problems on protondb are from people with AMD cards (which deck also has)

**Comment by michaelneverwins** [comment]
>Do you have an Nvidia GPU in your PC?

I have an AMD card, but it's a fairly old one (Radeon HD 7000 series).

**Comment by michaelneverwins** [comment]
I noticed the game is now verified for Proton 6.3-8 on Steam Deck — whereas, at the time of my previous posts, it was configured for Proton 7.0-3 by default — so I assume Valve re-tested it. I had actually been meaning to try again with Proton 7.0 since finding [this Steam forum thread](https://steamcommunity.com/app/1675200/discussions/1/5267542371383258908/), which suggests that simply rebooting the Steam Deck and avoiding the use of sleep mode before running the game might have gotten it working, but I never got around to it. I could still try it but, now that Valve has the game configured to run with Proton 6.3-8, I might as well go with that.

I just did a fresh install of the game (which I had removed after failing to get it working before), and ran it with Proton 6.3-8. Initially, there was no audio in cutscenes; then, when I restarted the Steam Deck and tried again, there was audio in cutscenes but no audio in gameplay. Maybe this doesn't always happen, given the odd inconsistency even in my own brief testing on one device, but I was definitely having audio problems that were not resolved by adjusting the volume. So I added `WINEDLLOVERRIDES="xaudio2_7=b,n" %command%` to the launch options, and now all audio is working.

This is the same tweak I had used on my Linux Mint PC when I'd had cutscene audio problems there (but, as noted in my first post, that was needed only for older Proton versions, while Proton 7.0 runs the game on my PC without any launch options; I've confirmed this is still the case with Proton 7.0-4 as of now).

In summary, the following works for me:
* Proton 6.3-8, with `WINEDLLOVERRIDES="xaudio2_7=b,n" %command%` in the launch options, on my Steam Deck.
* Proton 7.0-4, with no workarounds, on my Linux Mint 20.3 desktop PC.

**Comment by alasky17** [comment]
@michaelneverwins You are correct - there was a dxvk regression in the 7.0-x branch, so the game is set to prefer 6.3-8 to avoid this.  The regression is fixed in experimental though -- could you try retesting the game with experimental on your deck and seeing if you get any audio issues without the DLL override?  You can force just the one game to use Proton Experimental by going to settings -> compatibility for the game and then using the drop-down menu to select Proton Experimental.

**Comment by michaelneverwins** [comment]
@alasky17,
I just tested the game with Proton Experimental on my Steam Deck, and it does work without the Wine DLL override.

One more note on Proton 6.3-8 though: Before switching to Proton Experimental, I removed the DLL override launch option and tried running the game with Proton 6.3-8 again, just to confirm that the audio problems would come back, but the game wouldn't launch at all. I tried twice; it just gave me a black screen for a second and closed. So it looks like omitting the DLL override with Proton 6.3-8 can also crash the game, in addition to causing audio problems, but only sometimes. Or maybe the problem is that I had used the override and then removed it. I deleted the game's Proton files and tried with Proton 6.3-8 again, and it launched successfully but without audio in cutscenes, as expected.

After switching to Proton Experimental (still without any launch options), audio and graphics are both fine. I also deleted the Proton files again for good measure and launched with Proton Experimental again, and everything still works.

**Comment by alasky17** [comment]
@michaelneverwins Thank you for the update!  It sounds like experimental is the better default over 6.3-8 by a long shot :)

**Comment by XenoSlug** [comment]
Alan Wake crashes at the Remedy loading screen with Proton Experimental v9 and also with the Proton Hotfix release for apex legends. It works with Proton v8.

**Comment by kisak-valve** [comment]
Hello @XenoSlug, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the crash, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by XenoSlug** [comment]
[steam-108710.log](https://github.com/ValveSoftware/Proton/files/14547290/steam-108710.log)
and here is the [gist](https://gist.github.com/XenoSlug/b4f0a7bfa20b6003dcc581201cda3f68)

**Comment by alasky17** [comment]
@XenoSlug Thank you for reporting this!  It turns out that this due to the switch to enable nvapi by default.  The game does not like nvapi to be enabled on AMD.  This will be disabled by default whenever the next 9.0 releases go live.

**Comment by lucasreis1** [comment]
I'm experiencing some very strong texture flickerings in this game, both on X11 and Wayland compositors. Tested on Proton GE, Experimental, 8 and 7.0. Seems to be reduced on Proton 7, but still present.

**Comment by ghost** [comment]
There is a regression in Proton Experimental. Enemies are rendered transparent when illuminated by a flashlight. This does not occur in stable Proton releases, nor does it happen with Proton 11 Beta.

This should not be confused with any in-game visual effects. It is clearly not an intended game effect, as enemies become completely transparent and then suddenly return to normal.

This might be related to MSAA anti-aliasing. I've noticed several graphics-breaking glitches in Proton Experimental when MSAA is enabled in older games that only support MSAA (For example, Just Cause 2 has similar transparency issues, with the player character sometimes turning completely transparent when MSAA is enabled). However, Alan Wake does not provide an option to disable MSAA, so I can't test whether that is the cause in this case.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/bbcc233c-c69d-4331-9278-530f40d39885" />

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/4a10c6f1-856a-4fee-b037-71aee950721c" />

**Comment by kisak-valve** [comment]
Hello @Keenwhisk, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by ghost** [comment]
> Hello [@Keenwhisk](https://github.com/Keenwhisk), please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Hello

https://gist.github.com/Keenwhisk/ef1f3e5cab546c202c9662f2fff48d35

[steam-108710.log](https://github.com/user-attachments/files/29342117/steam-108710.log)
